app/controllers/api/application_plans_controller.rb

```diff
@@ -45,26 +40,10 @@ def masterize
   protected
 
   def collection
-    scope = current_account
-
-    if params[:service_id].present?
-      scope = scope.accessible_services.find(params[:service_id])
-    end
-
-    scope.application_plans.includes(:issuer)
+    (@service || current_account).application_plans.includes(:issuer)
```

Comment:
It would be better if it is a method instead of `@service`

```suggestion
    context_object.application_plans.includes(:issuer)
```